apps/web/src/types/Symptoms.ts

```diff
@@ -0,0 +1,16 @@
+import { Severity } from './Severity';
+
+export class Symptoms {
+  fever!: Severity;
```

Comment:
why not? by default it would be false

```suggestion
  fever!: Severity | boolean = false;
```